plugins/performance-lab/includes/admin/load.php

```diff
@@ -471,3 +471,32 @@ function perflab_get_plugin_settings_url( string $plugin_slug ): ?string {
 
 	return null;
 }
+
+/**
+ * Prints the Performance Lab install notice after each feature plugin's row meta.
+ *
+ * @since n.e.x.t
+ *
+ * @param string $plugin_file Plugin file.
+ */
+function perflab_print_row_meta_install_notice( string $plugin_file ): void {
+	if ( ! in_array( strtok( $plugin_file, '/' ), perflab_get_standalone_plugins(), true ) ) {
+		return;
+	}
+
+	$message = sprintf(
+		/* translators: placeholder is link to Performance Lab screen */
+		__( 'This plugin is installed by %s.', 'performance-lab' ),
+		sprintf(
+			'<a href="%s">%s</a>',
+			esc_url( add_query_arg( 'page', PERFLAB_SCREEN, admin_url( 'options-general.php' ) ) ),
+			__( 'Performance Lab', 'performance-lab' )
+		)
```

Comment:
An alternative:

```suggestion
		/* translators: %s: link to Performance Lab settings screen */
		__( 'This plugin is installed by <a href="%s">Performance Lab</a>', 'performance-lab' ),
		esc_url( add_query_arg( 'page', PERFLAB_SCREEN, admin_url( 'options-general.php' ) ) ),
```

Reply:
I had done this originally but then I refactored it to extract the plugin name into a separate string since it is already a standalone translation string. But you know best

Reply:
The effort required for translators is the same (1 new string), plus plugin names are usually not translated.
I‘d prefer this version here

Reply:
Ok cool. Could you re-suggest since there is a merge conflict now?